### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ssurbhi09**
Added how to contribute on readme!

### CODE REVIEW

**Comment by ssurbhi09** [comment]
Any issues?
Or it is ohk!
Then merge it thankyou

File: README.md [hunk 1]
```diff
@@ -88,3 +88,92 @@ git push -u origin <your_branch_name>
 **14.** Congrats !! You made your first PR 🥳.
 
 <p align="center"><img alt="Hooray" width=35% src="https://media.giphy.com/media/TdfyKrN7HGTIY/giphy.gif"></p>
+
+
+
+# 💥 How to Contribute
+
+[![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat-square)](https://github.com/mohit200008/FoodSaver20008/pulls)
```
Comment (aakankshabhende) [inline README.md]: Change from food saver to student portal in the repository name

File: README.md [hunk 2]
```diff
@@ -88,3 +88,92 @@ git push -u origin <your_branch_name>
...
+[![Open Source Love](https://badges.frapsoft.com/os/v1/open-source.png?v=103)](https://github.com/ellerbrock/open-source-badges/)
+
+- Take a look at the existing [Issues](https://github.com/ssurbhi09/Student-portal/issues) or [create a new issue](https://github.com/ssurbhi09/Student-portal/issues/new/choose)!
```
Comment (aakankshabhende) [inline README.md:99]: Change the username

File: README.md [hunk 3]
```diff
@@ -88,3 +88,92 @@ git push -u origin <your_branch_name>
...
+- [Fork the Repo](https://github.com/ssurbhi09/Student-portal/fork), create a branch for any issue that you are working on and commit your work.
+- Create a **[Pull Request](https://github.com/mohit200008/FoodSaver20008/compare)** (_PR_), which will be promptly reviewed and given suggestions for improvements by the community.
```
Comment (aakankshabhende) [inline README.md]: Change from food saver to student portal in the repository name and change the username

File: README.md [hunk 4]
```diff
@@ -88,3 +88,92 @@ git push -u origin <your_branch_name>
...
+
+## ⭐ HOW TO MAKE A PULL REQUEST:
+
+**1.** Start by making a fork the [**Student-portal**](https://github.com/Student-portal) repository. Click on the <a href="https://github.com/mohit200008/FoodSaver20008/fork"><img src="https://i.imgur.com/G4z1kEe.png" height="21" width="21"></a> symbol at the top right corner.
+
+**2.** Clone your new fork of the repository:
+
+```bash
+git clone https://github.com/<your-github-username>/FoodSaver20008
+```
```
Comment (aakankshabhende) [inline README.md:112]: Change from food saver to student portal in the repository name

File: README.md [hunk 5]
```diff
@@ -88,3 +88,92 @@ git push -u origin <your_branch_name>
...
+
+**3.** Set upstream command:
+
+```bash
+git remote add upstream https://github.com/mohit200008/FoodSaver20008.git
```
Comment (aakankshabhende) [inline README.md]: Change from food saver to student portal in the repository name and change the username

File: README.md [hunk 6]
```diff
@@ -88,3 +88,92 @@ git push -u origin <your_branch_name>
...
+[![Open Source Love](https://badges.frapsoft.com/os/v1/open-source.png?v=103)](https://github.com/ellerbrock/open-source-badges/)
+
+- Take a look at the existing [Issues](https://github.com/ssurbhi09/Student-portal/issues) or [create a new issue](https://github.com/ssurbhi09/Student-portal/issues/new/choose)!
+- [Fork the Repo](https://github.com/ssurbhi09/Student-portal/fork), create a branch for any issue that you are working on and commit your work.
+- Create a **[Pull Request](https://github.com/mohit200008/FoodSaver20008/compare)** (_PR_), which will be promptly reviewed and given suggestions for improvements by the community.
+- Add screenshots or screen captures to your Pull Request to help us understand the effects of the changes that are included in your commits.
+
+## ⭐ HOW TO MAKE A PULL REQUEST:
+
+**1.** Start by making a fork the [**Student-portal**](https://github.com/Student-portal) repository. Click on the <a href="https://github.com/mohit200008/FoodSaver20008/fork"><img src="https://i.imgur.com/G4z1kEe.png" height="21" width="21"></a> symbol at the top right corner.
```
Comment (aakankshabhende) [inline README.md]: Change from food saver to student portal in the repository name and change the username

**Review by aakankshabhende — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (ssurbhi09) [reply README.md:99]: Jo Maine fork ki h wo ta Mohit wali?

**Comment by ssurbhi09** [comment]
All done mam @aakankshabhende if there is any issue left please let me know thankyou!

**Review by aakankshabhende — APPROVED** [review]

**Comment by aakankshabhende** [comment]
Good work :) @ssurbhi09

**Comment by ssurbhi09** [comment]
@aakankshabhende mam u didn't merge it why?

**Comment by aakankshabhende** [comment]
One more mentor should accept this PR to merge this into the main branch

**Review by aakankshabhende — APPROVED** [review]

**Comment by shivshankar9** [comment]
@ssurbhi09 remove "how to get started part" not needed

**Comment by ssurbhi09** [comment]
@shivshankar9 sir how to wali line hatni h ya puri how to  get started hatani h?

**Comment by shivshankar9** [comment]
> @shivshankar9 sir how to wali line hatni h ya puri how to get started hatani h?

How to get started and how to contribute both are same so remove old one

**Comment by ssurbhi09** [comment]
> > @shivshankar9 sir how to wali line hatni h ya puri how to get started hatani h?
> 
> How to get started and how to contribute both are same so remove old one

Sir now I don't see my previous commits that I have change.

**Comment by shivshankar9** [comment]
> > > @shivshankar9 sir how to wali line hatni h ya puri how to get started hatani h?
> > 
> > 
> > How to get started and how to contribute both are same so remove old one
> 
> Sir now I don't see my previous commits that I have change.

Just fetch main repository and commit again

**Comment by aakankshabhende** [comment]
@ssurbhi09 please fetch main repository and commit again to get this PR merged with the main

**Pull request closed without merging**